Source/g_game.c

```diff
@@ -2789,14 +2806,7 @@ byte *G_ReadOptions(byte *demo_p)
     }
   else  // defaults for versions < 2.02
     {
-      int i;  // killough 10/98: a compatibility vector now
-      for (i=0; i < COMP_TOTAL; i++)
-	comp[i] = compatibility;
```

Comment:
So, we do not consider Boom's compatibility flag anymore?

Reply:
It's always zero in PrBoom+ demos:
https://github.com/coelckers/prboom-plus/blob/57750b80f9787f2fdb1b87a0dc84588fb66ab6da/prboom2/src/g_game.c#L3483-L3484

Reply:
But one line above there is `boom_compatibility_compatibility`.

Reply:
But we do not support `boom_compatibility_compatibility`. I've never seen them, but I guess we should keep `compatibility` for demo playback.